### DISCUSSION TITLE
codeql with submodule will only return one module result?

### DISCUSSION THREAD

**Discussion by ox1234** [github/codeql#9209] [Q&A] [answered]
When i analyze a submodule maven project, codeql only return one of them result, is that a bug or need some configuration?
Here have many submodule
![image](https://user-images.githubusercontent.com/32171650/169218088-5c4481b6-a370-4a05-a621-9fb1a6ea6052.png)
But here only return jeecms-common result:
![image](https://user-images.githubusercontent.com/32171650/169218144-85189ca7-6252-4425-8afd-7e4a715e4749.png)

**Comment by hvitved** [comment, marked as answer]
Are you sure that the source code in the other submodules are compiled as part of the build? CodeQL will only include `.java` files that are part of the build.

**Reply by ox1234** [reply]
Sorry, it's my fault, i forget to use mvn clean to delete previous compile result. Thanks for your help
